web/app/Repository/GameRepository.php

```diff
@@ -17,21 +17,9 @@ public function storeGame(array $post): bool
         $RA = $whitePlayer->getELO(); //Old ELO of White Player
         $RB = $blackPlayer->getELO(); //Old ELO of Black Player
         $K = 40;
-        if ($post["winner"] == 'white') {
-            $SA = 1;
-            $SB = 0;
-            $gameoutcome = "White";
-        }
-        elseif ($post["winner"] == 'black') {
-            $SA = 0;
-            $SB = 1;
-            $gameoutcome = "Black";
-        }
-        elseif ($post["winner"] == 'remis') {
-            $SA = 0.5;
-            $SB = 0.5;
-            $gameoutcome = "Remie";
-        }
+
+        [$SA, $SB] = $this->getScore($post['winner']);
```

Comment:
in php, we use a common way of naming variables and how should they look.
for variables, we prefer $camelCase. other people prefer $snake_case (we only use it in old code).
abbreviations can lead to misunderstandings or make code harder to read and understand.

Reply:
Ik thats why i am Making commands like: // Selecting Username White